modules/extensions/src/path-style/shaders.glsl.js

```diff
@@ -57,3 +57,27 @@ float round(float x) {
 `
   }
 };
+
+export const offsetShaders = {
+  inject: {
+    'vs:#decl': `
+attribute float instanceOffsets;
+`,
+    'vs:DECKGL_FILTER_SIZE': `
+  float offsetWidth = abs(instanceOffsets * 2.0) + 1.0;
+  size *= offsetWidth;
+`,
+    'vCornerOffset = offsetVec;': `
+  float offsetWidth = abs(instanceOffsets * 2.0) + 1.0;
+  vec2 offsetCenter = -instanceOffsets * (isCap ? perp : miterVec * miterSize) * 2.0;
+  vCornerOffset = vCornerOffset * offsetWidth - offsetCenter;
+`,
+    'fs:#main-start': `
+  float isInside;
+  isInside = step(-1.0, vPathPosition.x) * step(vPathPosition.x, 1.0);
+  if (isInside == 0.0) {
```

Comment:
Would this be easier to understand?:
```glsl
if (vPathPosition.x < -1.0 || vPathPosition.x > 1.0) {
  discard;
}
```
or even this?:
```glsl
bool isInside = vPathPosition.x < -1.0 || vPathPosition.x > 1.0;
```

Reply:
I need to add manual anti-aliasing here later (waiting for a change in luma) so ` isInside` can be between 0 and 1, in which case it will be multiplied with the output alpha.

Reply:
Ahhh gotcha